plugins/woocommerce/src/Internal/Admin/Settings/Payments.php

```diff
@@ -667,16 +673,21 @@ function ( $a, $b ) {
 			// If the payment method has a description, sanitize it before use.
 			if ( ! empty( $recommended_pm['description'] ) ) {
 				$standard_details['description'] = $recommended_pm['description'];
-				// Make sure that if we have HTML tags, we only allow a limited set of tags.
+				// Make sure that if we have HTML tags, we only allow stylistic tags.
```

Comment:
I don't follow how `<a>` is a stylistic tag.

Reply:
It is not. It is the exception. I will update the comment.